package.json

```diff
@@ -180,6 +183,21 @@
                 "title": "Create New Deployment Slot",
                 "category": "Azure App Service"
             },
+            {
+                "command": "diagnostics.EnableRemoteDebug",
+                "title": "Enable Remote Debugging",
+                "category": "Azure App Service"
+            },
+            {
+                "command": "diagnostics.DisableRemoteDebug",
+                "title": "Disable Remote Debugging",
+                "category": "Azure App Service"
+            },
+            {
+                "command": "diagnostics.StartRemoteDebug",
```

Comment:
nit: I'd prefer if the command name was more similar to the title "diagnostics.AttachDebugger"